core/services/bulletprooftxmanager/eth_confirmer.go

```diff
@@ -393,6 +393,12 @@ func (ec *ethConfirmer) saveFetchedReceipts(ctx context.Context, receipts []Rece
 // In this case we mark these transactions as 'confirmed_missing_receipt' to
 // prevent gas bumping.
 //
+// FIXME: We should continue to attempt to resend eth_txes in this state on
```

Comment:
This will be fixed in a followup - see https://www.pivotaltracker.com/story/show/177389604